lib/registrant.rb

```diff
@@ -0,0 +1,45 @@
+class Registrant
+    
+    attr_reader :name,
+                :age,
+                :permit,
+                :license_data,
+                :take_written_test
```

Comment:
This should not be in the attr_reader, we dont' have an attribute called `@take_written_test` that we're trying to read.